src/pages/_app.jsx

```diff
@@ -4,12 +4,17 @@ import 'bootstrap/dist/css/bootstrap.css';
 
 const Application = ({Component, pageProps}) => (
   <>
-    <nav className='navbar navbar-dark bg-primary'>
+    <nav className='navbar navbar-expand-lg sticky-top navbar-dark bg-primary'>
```

Comment:
I think this whole navbar is going to be complex enough so that you could extract this into it's own function. Your call though, it's a very small thing anyways, but you could do something like this:

1. Define a new react component before the `Application` function:
```jsx
const Header = () => (
  <nav className='...'>
    // ...
  </nav>
);
```

2. Use that component in the main `Application` function, like this:
```jsx
const Application = ({Component, pageProps}) => (
  <>
    <Header />
    // ...
  </>
);
```